docs/adv/troubleshooting/test_command.md

```diff
@@ -0,0 +1,161 @@
+- The process might have died.
+
+## Beacon
+
+#### Ping
+
+- Beacon node might not be started or is not reachable.
+
+#### PingMeasure
+
+- Beacon node might be too far away (geographically) from you.
+- Your network latency might be high. Verify with the `charon test infra` tests.
+
+#### Version
+
+- The beacon node version is not compatible with charon.
+
+#### IsSynced
+
+- Beacon node is not synced to the network.
+
+#### PeerCount
+
+- Beacon node does not have enough peers. This may result in slower fetching and broadcasting of slots and duties.
+
+#### PingLoad
+
+This is a load test, to enable it add the `--load-test` flag.
+
+Same causes as PingMeasure test apply here.
+
+#### Simulation
+
+This is a load test, to enable it add the `--load-test` flag.
+
+Same causes as PingMeasure test apply here and additionally:
+
+- The infrastructure on which the beacon node run (amount of RAM, disk IOPS) might be not enough to hold the number of validators supplied.
+
+## Validator
+
+#### Ping
+
+- Validator client might not be started or is not reachable.
+
+#### PingMeasure
+
+- Validator client might be too far away (geographically) from you. It is expected to run the validator client on the same machine.
+
+#### PingLoad
+
+Same causes as PingMeasure test apply here.
+
+## MEV
+
+#### Ping
+
+- MEV relay might not be started or is not reachable.
+
+#### PingMeasure
+
+- MEV relay might be too far away (geographically) from you.
+- Your network latency might be high. Verify with the `charon test infra` tests.
+
+#### CreateBlock
+
+Same causes as PingMeasure test apply here and additionally:
+
+- MEV relay might be too slow in block production.
+
+#### CreateMultipleBlocks
+
+Same causes as CreateBlock test apply here and additionally:
+
+## Infra
+
+#### DiskWriteSpeed
+
+- Your SSD/HDD write speed is too low. Minimum write speed is 500MBs, recommended write speed is 1000MBs.
+
+#### DiskWriteIOPS
+
+- Your SSD/HDD write operations per second are too low. Minimum write IOPS are 1000, recommended write IOPS are 2000.
+
+#### DiskReadSpeed
+
+- Your SSD/HDD read speed is too low. Minimum read speed is 500MBs, recommended read speed is 1000MBs.
+
+#### DiskReadIOPS
+
+- Your SSD/HDD read operations per second are too low. Minimum read IOPS are 1000, recommended read IOPS are 2000.
+
+#### AvailableMemory
+
+- Your available memory (RAM) is not enough. Minimum available memory should be 2GB, recommended available memory is 4GB. Note that this test is best possible estimation, as memory availability is hard to be predicted, especially if the command is ran in a virtualised environment (i.e.: Docker container).
+
+#### TotalMemory
+
+- Your total memory (RAM) is not enough. Minimum total memory should be 4GB, recommended total memory is 8GB.
+
+#### InternetLatency
+
+- Your internet latency to the nearest server is too high. Latency is expected to be at least less than 50ms and at best less than 20ms.
```

Comment:
```suggestion
- Your internet latency to the nearest test server is too high. Latency is expected to be at least less than 50ms and at best less than 20ms.
```

How will this hold up in apac? what provider are we using for the test servers?

Reply:
We are using Ookla.